File: app/src/main/java/org/antrack/app/ui/fragments/ImageActivity.java
package org.antrack.app.ui.fragments;

import android.app.Activity;
import android.content.Intent;
import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.os.Bundle;
import android.util.DisplayMetrics;
import android.widget.ImageView;

import org.antrack.app.libs.Images;

import app.R;

public class ImageActivity extends Activity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_image);

        Intent i = getIntent();
        String path = i.getExtras().getString("path");

        ImageView imageView = (ImageView) findViewById(R.id.activity_image_image);
        DisplayMetrics metrics = Images.getDisplayMetrics(this);
        Bitmap bitmap = Images.decodeFile(path, metrics.widthPixels);
        imageView.setImageBitmap(bitmap);
    }
}

File: app/src/main/java/org/antrack/app/ui/fragments/ImageActivity.kt
package org.antrack.app.ui.fragments

import android.app.Activity
import android.os.Bundle
import android.widget.ImageView
import app.R
import org.antrack.app.libs.Images

class ImageActivity : Activity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_image)

        val i = intent
        val path = i.extras.getString("path")

        val imageView = findViewById(R.id.activity_image_image) as ImageView
        val metrics = Images.getDisplayMetrics(this)
        val bitmap = Images.decodeFile(path, metrics.widthPixels)
        imageView.setImageBitmap(bitmap)
    }
}

